virtual_rainforest/core/data.py

```diff
@@ -0,0 +1,362 @@
+"""API documentation for the :mod:`core.data` module.
+**************************************************
+
+This module handles the population and storage of data sources used to run Virtual
+Rainforest simulations.
+
+The Data class
+==============
+
+The core :class:`~virtual_rainforest.core.data.Data` class is a wrapper around an
+:class:`~xarray.DataSet` object, which is used to core variables in the simulation on
+shared axes. The :class:`~xarray.DataSet` object is stored as the :attr:`Data.data`
+attribute, and this attribute can be used directly to access :class:`~xarray.DataSet`
+methods.
+
```

Comment:
When you explained this in person it made a lot of sense, however, the way it is written here is a bit difficult to understand. Do we need this level of detail here? If yes, how can we make it more accessible?

Reply:
This is the API documentation for developers - which is traditionally quite terse 😄  - where my verbal description and the user docs are supposed to give a more general overview. That doesn't mean this text can't be improved, but the detail is deliberate.

Reply:
I've expanded this section and tried to make it clearer.